.github/workflows/build.yaml

```diff
@@ -1,8 +1,7 @@
-name: Build
+name: Build ROCK
```

Comment:
I would use "OCI" everywhere... as some images are docker built and some are rockcraft one... while both are OCI.